gobblin-data-management/src/main/java/gobblin/data/management/copy/replication/ConfigBasedDataset.java

```diff
@@ -89,9 +89,7 @@ private void calculateDatasetURN(){
       } catch (IOException e1) {
         // ignored
       }
-    }
-
-    this.datasetURN = e.toString();
+    } else this.datasetURN = e.toString();
```

Comment:
@ibuenros  
Regarding to the second comments above: 
I agree. The main reason for implementing black/whitelist in this level: 
- There's no base Implementation of `DatasetsFinder`, each kinds of dataset finder is implementing the interface, Which means it is hard to extending black/whitelist into every dataset finder.  For this particular example, I could choose to implement this function in `configBasedDatasetsfinder`.  
- The current implementation is equivalent to put a filter when creating `CopyableDatasetRequestor`, which is on dataset-level but not specific to a type of dataset.